cluster-autoscaler/metrics/metrics.go

```diff
@@ -235,6 +235,14 @@ var (
 		},
 	)
 
+	oldUnregisteredNodesRemovedCount = k8smetrics.NewCounter(
+		&k8smetrics.CounterOpts{
+			Namespace: caNamespace,
+			Name:      "old_unregistered_nodes_removed_count",
```

Comment:
`Help` on the next line doesn't explain why you have an `old_` prefix in here.  Why is this not just `unregistered_nodes_removed_count`?

Reply:
that was inspired by the way the variable is named inside the autoscaler.